oshi-core/src/main/java/oshi/hardware/platform/unix/solaris/SolarisSoundCard.java

```diff
@@ -47,28 +47,27 @@ public SolarisSoundCard(String kernelVersion, String name, String codec) {
         vendorMap.clear();
         productMap.clear();
         List<String> sounds = new ArrayList<>();
-        String key = "";
+        String key = ""; 
         for (String line : ExecutingCommand.runNative(LSHAL)) {
             if (line.startsWith("udi =")) {
-                // we have the key.
+        	    // we have the key.
```

Comment:
This looks suspiciously like you replaced four spaces with a tab.  :)

Reply:
That was not intentional, i was just used to using tabs :)
I fixed it.

Reply:
Not a big deal, formatters fix it and other whitespace as part of the maven build.

Thanks for your contribution to Open Source!